stories/pdf-viewer/pdf-viewer.stories.js

```diff
@@ -0,0 +1,148 @@
+import '../../dist/dialog.css';
+import '../../dist/pdf-viewer.css';
+import '../../dist/message-page.css';
+import '../../dist/title.css';
+
+export default {
+    title: 'Components/PDF Viewer',
+    parameters: {
+        description: `
+The PDF viewer component can be either displayed within a page or inside of a dialog.
+
+##Usage
+**Use the PDF viewer if:**
+
+- You want your app to display PDF files on all devices and platforms.
+- You want the users of your app to be able to preview their documents as PDF files right beside your app.
+- You need to ensure the consistent behavior of PDF files across your app.
+
+**Do not use the PDF viewer if:**
+
+- You need to provide an interactive PDF file (such as a data input form).
+
+`,
+        tags: ['f3', 'a11y']
+    }
+};
+
+export const dialog = () => `
+<div class="fd-dialog-docs-static fd-pdf-viewer fd-dialog fd-dialog--active">
+    <div class="fd-dialog__content fd-dialog__content--l" role="dialog" aria-modal="true" aria-labelledby="dialog-title-1">
+        <header class="fd-dialog__header fd-bar fd-bar--header">
+            <div class="fd-bar__left">
+                <div class="fd-bar__element">
+                    <h2 class="fd-title fd-title--h5" id="dialog-title-1">
+                        PDF Viewer title
+                    </h2>
+                </div>
+            </div>
+        </header>
+        <div class="fd-dialog__body">
+            <iframe
+              class="fd-pdf-viewer__content"
+              title='PDF viewer'
+              src="https://sapui5.hana.ondemand.com/test-resources/sap/m/demokit/sample/PDFViewerEmbedded/sample.pdf#view=FitH"}
+              frameBorder='0'></iframe>
+        </div>
+        <footer class="fd-dialog__footer fd-bar fd-bar--footer">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button class="fd-dialog__decisive-button fd-button fd-button--compact">
+                        Download
+                    </button>
+                </div>
+                <div class="fd-bar__element">
+                    <button class="fd-dialog__decisive-button fd-button fd-button--compact">
+                        Close
+                    </button>
+                </div>
+            </div>
+        </footer>
+    </div>
+</div>
+`;
+
+dialog.storyName = 'Dialog';
+dialog.parameters = {
+    docs: {
+        iframeHeight: 400,
+        storyDescription: 'When displayed in a dialog, download and close buttons are in the footer. The PDF Viewer dialog does not support resizing. For mobile, ensure the modal takes up the entire screen by using the <code>.fd-dialog__content--mobile</code> class.'
```

Comment:
![Screen Shot 2020-09-04 at 12 24 01 PM](https://user-images.githubusercontent.com/2471874/92273918-849d3380-eea9-11ea-8832-9e9aff8ee262.png)

Do we need to escape the double underscores?  Not sure why it is showing up like this for me

Reply:
I've removed this part of the description